small/matras.c

```diff
@@ -335,6 +335,33 @@ matras_dealloc_range(struct matras *m, matras_id_t range_count)
 	matras_dealloc(m);
 }
 
+/**
+ * Reserve the max amount of extents required to successfully touch @p count
+ * blocks. The extents are reserved in the allocator given on construction.
+ */
+int
+matras_touch_reserve(struct matras *m, unsigned count)
+{
+	/* No `matras_touch` calls planned. */
```

Comment:
And bring back `assert(count >= 0)` here.

Reply:
Asserted here and in the `matras_allocator_reserve`.